modules/extension/wmts/src/test/java/org/geotools/data/wmts/client/WMTSTransparentTileTest.java

```diff
@@ -0,0 +1,125 @@
+package org.geotools.data.wmts.client;
+
+import org.geotools.data.wmts.model.TileMatrixSet;
+import org.geotools.data.wmts.model.WMTSCapabilities;
+import org.geotools.data.wmts.model.WMTSLayer;
+import org.geotools.data.wmts.model.WMTSServiceType;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.map.Layer;
+import org.geotools.map.MapContent;
+import org.geotools.renderer.GTRenderer;
+import org.geotools.renderer.lite.StreamingRenderer;
+import org.geotools.swing.SingleLayerMapContent;
+import org.geotools.tile.util.AsyncTileLayer;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.opengis.referencing.FactoryException;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.Rectangle;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static org.geotools.data.wmts.client.WMTSTileFactory4326Test.createCapabilities;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+/**
+ * @author peck7 on 21/12/2017.
+ */
+public class WMTSTransparentTileTest {
+
+    WMTSTileService service;
+
+    @BeforeClass
+    public static void init() {
+        Logger LOGGER = org.geotools.util.logging.Logging.getLogger("org.geotools");
+        LOGGER.setLevel(Level.FINE);
+    }
+
+    @Before
+    public void setup() throws Exception {
+        service = createKVPService();
```

Comment:
Do you need an `@After` `tearDown` to dispose this service?

Reply:
not sure, but added